sdk/core/abort-controller/package.json

```diff
@@ -47,14 +47,14 @@
     "url": "git+https://github.com/Azure/azure-sdk-for-js.git"
   },
   "keywords": [
-    "Azure",
-    "Aborter",
-    "AbortSignal",
-    "Cancellation",
-    "Node.js",
-    "TypeScript",
-    "JavaScript",
-    "Browser"
+    "azure",
+    "aborter",
+    "abortSignal",
```

Comment:
Several of these got camelCased instead of lowercased. Same with core-logger and storage packages.

Reply:
done.